views/tables.ejs

```diff
@@ -49,6 +49,23 @@
         window.alert(`There was an error when attempting to delete the table:\n\n${err}`)
       })
     }
+
+    function purgeTable(TableName) {
+      if(!confirm(`Are you sure you want to purge all records from table "${TableName}" ?`)){
+        return
+      }
+
+      fetch(`/tables/${TableName}/all`, {
+        method: 'delete'
+      }).then((response) => {
+        if (!response.ok) {
+          throw new Error
+        }
+        window.location.href = '/'
```

Comment:
location.reload() is probably better. Than it doesn't matter from which page that action was triggered.